packages/documentation/src/pages/getting-started/internal-tools.mdx

```diff
@@ -213,6 +194,9 @@ SOCKS proxy server is running. You can `$ nc -z 127.0.0.1 2001` as a first step.
 1. Check your connection by navigating to Sentry at
    http://sentry.vetsgov-internal.
 
+## Issues
+If your key doesn't seem to be working, contact the Platform team in the *#vetsgov-devops* Slack channel.
+
 ## Tools
```

Comment:
I think we should move this tools section into another document and link to it. Almost like we should have some guide to each one of these tools in the documentation.